configuration/micro_kernel_trait.rst

```diff
@@ -205,6 +212,39 @@ Before continuing, run this command to add support for the new dependencies:
 
     $ composer require symfony/yaml symfony/twig-bundle symfony/web-profiler-bundle doctrine/annotations
 
+Next, create a new extension class that defines your app configuration and
```

Comment:
To be honest, I'm not really sure if this is a good place to document `AbstractExtension`. What is its use-case when using MicroKernelTrait?

If we want to keep it, I would suggest moving the changes in this document to a new section below this one. Currently, I'm afraid the doc can make people think that they must use the app extension (or `build()` method) when using the micro kernel.

Reply:
> To be honest, I'm not really sure if this is a good place to document AbstractExtension. What is its use-case when using MicroKernelTrait?

Apps on bundle-less approach still need to define custom configuration, and it's only possible through a new extension, even if the kernel class can implement the ExtensionInterface it's better now by creating a new extension class and extending from AbstractExtension.

> If we want to keep it, I would suggest moving the changes in this document to a new section below this one. Currently, I'm afraid the doc can make people think that they must use the app extension (or build() method) when using the micro kernel.

I think you're right, it could be better in a new section with the proper explanation.